core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java

```diff
@@ -192,7 +192,7 @@ public void start() throws IOException {
                       LOG.warn("IOException encountered during active sync while starting {}", e);
                     }
                   }
-          )).get();
+          ));
```

Comment:
@yuzhu just to make sure. operations on the sub-tree being synced are still blocked because of locking correct? by not waiting we are allowing operations on other parts of the tree which are in sync to proceed?

Reply:
Yes, that is correct.